File: codeWars/binary_multiple_of_3/binary_multiple_of_3.py
# Binary multiple of 3

import re


def binary_multiple_of_3(str):
    regex = r"^(0*11*|0*)$"
    return bool(re.match(regex, str))


print(binary_multiple_of_3("000"))  # True
print(binary_multiple_of_3("001"))  # False
print(binary_multiple_of_3("011"))  # True
print(binary_multiple_of_3("110"))  # True
print(binary_multiple_of_3(" abc "))  # False

File: codeWars/katas/binary_multiple_of_3/binary_multiple_of_3.js
// Binary multiple of 3

function binaryMultipleOf3(str) {
    const regex = /^(0*11*|0*)$/;
    return regex.test(str);
}

console.log(binaryMultipleOf3('000'));  // Output: true
console.log(binaryMultipleOf3('001'));  // Output: false
console.log(binaryMultipleOf3('011'));  // Output: true
console.log(binaryMultipleOf3('110'));  // Output: true
console.log(binaryMultipleOf3(' abc '));  // Output: false